doc/api/stream.md

```diff
@@ -1506,13 +1506,67 @@ async function print(readable) {
 print(fs.createReadStream('file')).catch(console.error);
 ```
 
-If the loop terminates with a `break` or a `throw`, the stream will be
-destroyed. In other terms, iterating over a stream will consume the stream
+If the loop terminates with a `break`, `return` or a `throw`, the stream will
```

Comment:
Aren't those just all the ways to terminate a loop?

Reply:
Yeah, I just added `return` as it was missing IMO

Reply:
Our style guide calls for use of serial commas:

```suggestion
If the loop terminates with a `break`, `return`, or a `throw`, the stream will
```